app/controllers/api/v1/items_controller.rb

```diff
@@ -28,8 +28,6 @@ def find_all
       render json: ItemSerializer.new(items)
     rescue ArgumentError => exception
       render json: ErrorSerializer.format_error(exception, "400"), status: :bad_request
-    rescue StandardError => exception
```

Comment:
Thank you for catching this, I must have missed this error code specification in the criteria.